src/fmrib/NeuroPM/README.md

```diff
@@ -1,13 +1,52 @@
-### This is guidance for new users of the CCRF-adapted Matlab version of the cTI methodology
-##### Put 3 model input files into the "/io" folder;
+## Loading Dataset
+
+### Ensure loading the inputs in the correct format. The code applies the following,
+
+```
+1. Load the dataset file in .csv format (pre-processed through "preprocess_data_preparation.R")
+2. Convert the dataset from tabular to numerical format.
+The dataset is represented as [No.Subjects by No.Features], that is [27,439 by 1,073]. 
+3. Load the labels file in .csv format as [No.Subjects by 4]
+The labels files include 4 columns of information for every subject (27,439 rows). 
+Labels corresponding to the blood pressure categorisation are stored in the 4th column
+```
+
```

Comment:
You dont have to say exactly the number.
Can u just say something along the lines of "ukb_feature_select.csv" is [No.Subjects by No.Features]
"label_feature_select.csv" is [No.Subjects] where "bp_group" is the column name we want to use

Basically imagine if someone who doesnt know how to program is given this code, they need to know exactly what is the inputs, or what to change in the code for their own data. in this case the parts they need to change are the two input file names, the column name of the label, and the numerical value representing the label (0/1/2).

Reply:
It would be great if u could just tell ppl you need to change "XX" on line "XX", and the data has to be "XX by XX"